model_path: /harddisk/jcaiaq/dataset/cpd/models/20220421_p&b_cpn_32_34188.pth
gripper_depth: 0.10327